load-tests/load-tester/src/main/java/io/camunda/zeebe/Starter.java

```diff
@@ -53,16 +68,48 @@ public class Starter extends App {
 
   @Override
   public void run() {
+    partitionToTimerMap = new ConcurrentHashMap<>();
+    responseLatencyTimer =
+        MicrometerUtil.buildTimer(StarterLatencyMetricsDoc.RESPONSE_LATENCY).register(registry);
+    results = new CopyOnWriteArrayList<>();
+    partitionToTimerMap.put(
+        1,
+        MicrometerUtil.buildTimer(StarterLatencyMetricsDoc.DATA_AVAILABILITY_LATENCY)
+            .tag("partition", "1")
+            .register(registry));
+    partitionToTimerMap.put(
+        2,
+        MicrometerUtil.buildTimer(StarterLatencyMetricsDoc.DATA_AVAILABILITY_LATENCY)
+            .tag("partition", "2")
+            .register(registry));
+    partitionToTimerMap.put(
+        3,
+        MicrometerUtil.buildTimer(StarterLatencyMetricsDoc.DATA_AVAILABILITY_LATENCY)
+            .tag("partition", "3")
+            .register(registry));
     final CamundaClient client = createCamundaClient();
 
     // init - check for topology and deploy process
     printTopology(client);
     deployProcess(client, starterCfg);
 
     // setup to start instances on given rate
+    piCheckExecutorService = Executors.newScheduledThreadPool(20);
     final CountDownLatch countDownLatch = new CountDownLatch(1);
-    final ScheduledExecutorService executorService =
-        Executors.newScheduledThreadPool(starterCfg.getThreads());
+
+    piCheckExecutorService.scheduleAtFixedRate(
+        () -> {
+          try {
+            queryProcessInstances(client);
+          } catch (final Exception e) {
+            LOG.error("Failed to query process instances. Will retry...", e);
+          }
+        },
+        1000,
```

Comment:
🔧 these params should be configurable. 

Is it expected that there is no data before 1 second is passed? Is that related to ES index refresh?